UserCard.jsx

```diff
@@ -0,0 +1,39 @@
+function UserCard({ userId }){
+  const [user, setUsers] = React.useState([]);
```

Comment:
A couple of small notes here:

1. Since this piece of state is representing a single user, I think it makes more sense for the initial value to be `null` or an empty object instead of an array. Setting the initial value as an array signals to other developers to expect a collection of many things. 
2. Along those same lines, I would name the function `setUser` instead of `setUsers` - it's a small change, but helps to communicate what the code is doing.

Reply:
Yea that makes a lot of sense, will also refactor this, thank you.